lib/form/form.tsx

```diff
@@ -83,6 +86,15 @@ function FormComp<T extends Record<string, any> = Record<string, any>>(
     }, [] as string[]);
   }, [formFieldsRef]);
 
+  const getErrorsMap = useCallback(() => {
+    const errorsMap: Record<string, string[]> = {};
+    formFieldsRef.current.forEach((field) => {
+      const name = field.props.name;
+      errorsMap[name] = field.errors;
```

Comment:
Curious what you think about the idea of doing a `fillPath` here instead of `errorsMap[name]`.

That way, instead of:

```js
errorsMap = {
     'one.two': ["No"]
}
```

It'd be more like:

```js
errorsMap = {
     one: {
         two: ["No"]
     }
}
```

I _think_ this might reduce the number of support tickets, open new workflows, and more... But it'd be more computationally expensive.

Curious what you think - I'm not hard sold one way or another.